pageserver/src/tenant/size.rs

```diff
@@ -178,21 +186,28 @@ pub(super) async fn gather_inputs(
     // our advantage with `?` error handling.
     let mut joinset = tokio::task::JoinSet::new();
 
-    let timelines = tenant
+    tenant
         .refresh_gc_info()
         .await
         .context("Failed to refresh gc_info before gathering inputs")?;
 
-    if timelines.is_empty() {
-        // All timelines are below tenant's gc_horizon; alternative would be to use
-        // Tenant::list_timelines but then those gc_info's would not be updated yet, possibly
-        // missing GcInfo::retain_lsns or having obsolete values for cutoff's.
-        return Ok(ModelInputs {
-            updates: vec![],
-            retention_period: 0,
-            timeline_inputs: HashMap::new(),
-        });
-    }
+    let timelines = tenant.list_timelines();
+
+    // this is an actual metric changing change, commenting this part out:
+    //
+    // if timelines.is_empty() {
+    //     // All timelines are below tenant's gc_horizon; alternative would be to use
+    //     // Tenant::list_timelines but then those gc_info's would not be updated yet, possibly
+    //     // missing GcInfo::retain_lsns or having obsolete values for cutoff's.
+    //     //
+    //     // FIXME: timelines which are under gc_horizon do have about 32MB of logical_size and so
+    //     // they will also have tenant_size, so perhaps this is the wrong way?
+    //     return Ok(ModelInputs {
+    //         updates: vec![],
+    //         retention_period: 0,
+    //         timeline_inputs: HashMap::new(),
+    //     });
+    // }
```

Comment:
Still need to discuss this. This change is not absolutely necessary, we could leave this in and solidify `tenant_size: 0` behaviour which was an annoyance in early phases.

Reply:
I prefer non-zero size. 
To me is really weird to see size=0 for database. It feels like something gone wrong.

Reply:
Very much agreed. Removed in latest push.